src/message-subscription.js

```diff
@@ -15,11 +15,13 @@ export class MessageSubscription {
     this.logger = options.logger
     this.connectionTimeout = options.connectionTimeout
     this.onMessageHook = options.onMessageHook
+    this.onMessageDeletedHook = options.onMessageDeleted
```

Comment:
I think this should be

```suggestion
    this.onMessageDeletedHook = options.onMessageDeletedHook
```

Would have been caught by the tests when the server PR is deployed I imagine.